vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSpaceController.java

```diff
@@ -66,10 +66,7 @@ public String space(@PathVariable("id") String id, Model model) {
             model.addAttribute("externalLinkList", externalLinkManager.getLinkDisplays(id));  
         }
         else {
-            model.addAttribute("showAlert", true);
-            model.addAttribute("alertType", "danger");
-            model.addAttribute("messageType","invalidSpace");
-            model.addAttribute("message", "Access Denied.");
+            return "badrequest";
```

Comment:
I would redirect to `/404` or `/403`. There will be other places where we wanna show that and having a dedicated page for that makes sense. Plus, Spring security tries to redirect to `/403` I believe and right now that just generates a 404 exception, so it would be nice to have anyways.